.buildkite/pipeline.yml

```diff
@@ -20,3 +20,8 @@ steps:
     plugins:
       'docker-compose#v1.7.0':
         run: fusion-cli
+  - name: ':jest:'
```

Comment:
For some reason the same tests have isolation concerns under tape, but work fine under jest. They're easy to move around later once we figure out the isolation problems.